andela/wg-millennium-falcon
# #157731579 Authors field as a foreign field

### What does this PR do?
Makes the Authors field for exercises a foreign field, instead of a single field.

### Description of Task to be completed?
Authors field for exercises should not be a single field, instead it should be a foreign field to improve the optimisation of the database and provide users with a drop down menu with all the authors available.

### How should this be manually tested?
* Run the application and login
* Add a new exercise and you should find a dropdown menu on the `License author` field with the available authors.
<img width="1197" alt="screen shot 2018-06-06 at 12 38 18" src="https://user-images.githubusercontent.com/26734746/41031750-2b0087f6-698a-11e8-8d32-4e223814c718.png">


### Any background context you want to provide?
Previously, the License Author field was a single field and not a foreign field.

### What are the relevant pivotal tracker stories?
[#157731579](https://www.pivotaltracker.com/story/show/157731579)
